DockerfileNEW

```diff
@@ -0,0 +1,80 @@
+# Stage: base image
+FROM node:18.12-bullseye-slim as base
+
+ARG BUILD_NUMBER=1_0_0
```

Comment:
In current app this is defined as below. Is this ok?

`ARG BUILD_NUMBER
ARG GIT_REF`